PTFE-1450 warn about failing queues (#168)
scality/bert-e: bert_e/templates/queue_build_failed.md

{% extends "message.md" %}

{% block title -%}
Queue build failed
{% endblock %}

{% block message %}

The corresponding build for the queue failed:

- Checkout the [status page]({{ frontend_url }})

> - Identify the failing build and review the logs.

Author: right, was contemplating if we should add this in or not. Guess we should 😅

> - Identify the failing build and review the logs.

Author: done

{% extends "message.md" %}

{% block title -%}
Queue build failed
{% endblock %}

{% block message %}

The corresponding build for the queue failed:

- Checkout the [status page]({{ frontend_url }}).
- Identify the failing build and review the logs.

> - If no issue is found, re-run the build.

Reviewer: or no need if there is a more recent green build on the same column.
But I don't know if we want to cover all cases here

Author: That's a good point, I would say let's start like this, and if users get confused we'll adapt. We know where the file is it's only a matter of editing this template

- If an issue is identified, checkout the steps below to remove

>   the pull request from the queue.

Reviewer: ```suggestion
  the pull request from the queue for further analysis and maybe rebase/merge.
```

<details>
  <summary><b>Remove the pull request from the queue</b></summary>

> - Add a `/wait` comment on the pull request.

Reviewer: ```suggestion
- Add a `/wait` comment on this pull request.
```

> - Click on login on the [status page]({{ frontend_url }})

Reviewer: ```suggestion
- Click on login on the [status page]({{ frontend_url }}).
```

- Go into the [manage]({{ frontend_url }}/manage) page.
- Find the option called `Rebuild the queue` and click on it.

>   This will remove the pull request from the queue.

Reviewer: ```suggestion
  This will remove loop again on all pull requests to put the valid ones in the queue again, but this will skip this one.
```

Author: ```suggestion
  Bert-E will loop again on all pull requests to put the valid ones
  in the queue again, while skipping the one with the `/wait` comment.
```

> - Wait for the new queue to merge, then update your pull request

Reviewer: ```suggestion
- Wait for the new queue to merge, then merge/rebase your pull request
```

>   with the latest change to then work on a proper fix.

Reviewer: ```suggestion
  with the latest changes to then work on a proper fix.
```
